lisa/operating_system.py

```diff
@@ -1394,6 +1394,24 @@ def capture_system_information(self, saved_path: Path) -> None:
             saved_path / "centos-release.txt",
         )
 
+    def _initialize_package_installation(self) -> None:
+        # refer https://www.centos.org/centos-linux-eol/
```

Comment:
As the document, it mentions only CentOS 8. How about 7.x?

Reply:
till now, I didn't find this issue on CentOS 7.